tests/instructions/run-local.md

```diff
@@ -0,0 +1,87 @@
+
+
+## Requires Linux
+
+Docker only supports IPV6 on Linux
+
+## Enable IPV6 in Docker
+
+Add or edit `/etc/docker/daemon.json`.  If you are creating a new file, then the file
+should contain this content.  If you are editing an existing file, then add this content
+to the existing content.  For more information, see the [Docker docs](https://docs.docker.com/config/daemon/ipv6/).
+
+```json
+{
+  "ipv6": true,
+  "fixed-cidr-v6": "2001:db8:1::/64"
+}
+```
+
+## Volumes to be mounted in the container
+
+There are four volumes mounted from your machine into the container:
+- The directory containing the ClickHouse `deb` files
+- The `tests` dir from the ClickHouse GitHub repo
+- The directory where test logs will be written to
+- The directory where server logs will be written to
+
+You should create the four directories and replace the left-hand side of the volume 
+statements with the path to your directories.
+
+## Script to launch the container
+
+TODO: Split run.sh into `install.sh` and `test.sh` so that the install can be done once, and 
+then individual tests can be run instead of all.
+
+```bash
+docker run -ti \
```

Comment:
(Cosmetic: argument `-it` instead of `-ti` would be slightly more idiomatic in Docker (i.e. mentioned more frequently in tutorials, guides etc.) but both are eventually doing the same)